### ISSUE TITLE
Calliope: name clashes (e.g. "sum")

### ISSUE THREAD

**Issue by rbudde** [OpenRoberta/robertalab#1172]
Using "sum" as variable name gives a redeclaration error (with the function "sum" from MicroBit.h:41) and the program cannot be compiled

**Expected behavior**
- this name clash and all other clashes of this kind should be avoided.
- if not possible, at least the compiler message has to be shown to the user (maybe he/she can then avoid the problem)

**General remark**
 - a clear concept of separating user-defined names and ora-defined names from "standard library" names is needed

**Comment by bjost2s** [comment]
same for "distance" (discovered) yesterday ;-)

**Comment by philippmaurer** [comment]
Same for "fdd" when using the 4-digit display on calliope. (Mentioned in a support ticket)

**Comment by boonto** [comment]
Yes, "fdd" and "ledBar" are clashing. I forgot to add the underscore to the variable declarations.

**Comment by boonto** [comment]
I fixed it for "fdd" and "ledBar", the others are still a problem.

**Comment by bjost2s** [comment]
We should prefix user defined names in general, no way to keep the reserved words list in blockly up to date.
